app/javascript/packs/base.js

```diff
@@ -4,6 +4,9 @@ import {
   getInstantClick,
   initializeTouchDevice,
 } from '../topNavigation/utilities';
+import { getFocusTrapToggle } from '../utilities/getFocusTrapToggle';
+
+window.getFocusTrapToggle = getFocusTrapToggle;
```

Comment:
This is purely so I can reference it in [`showModal`](https://github.com/forem/forem/pull/12511/files#diff-f6ec62b981153ef2ad9352f3a6ff688bdf8db52f000ed46333c3954b6ea88310) which is used across a lot of the older js initializers